src/windshaft/error.js

```diff
@@ -14,7 +14,7 @@ var WindshaftError = function (error) {
   }
   if (this.isAnalysisError(error.type)) {
     this.context = error.analysis && error.analysis.context;
-    this.analysisId = error.analysis && error.analysis.node_id;
+    this.analysisId = error.analysis && (error.analysis.node_id || error.analysis.id);
```

Comment:
@dgaubert do you know why this (:point_up:) might happen? Shouldn't `error.analysis.node_id` always be populated?